config/webpacker.yml

```diff
@@ -83,7 +83,7 @@ production:
   <<: *default
 
   # Production depends on precompilation of packs prior to booting for performance.
-  compile: false
+  compile: true
```

Comment:
Making some notes of things I've tried that have not worked. Changing this flag to true did not make the Travis build succeed. So far, can't tell if I should revert this or leave this.